dist/index.js

```diff
@@ -148,7 +148,13 @@ async function runScan({ source, failBuild, severityCutoff, outputFormat }) {
 
   failBuild = failBuild.toLowerCase() === "true";
 
-  cmdArgs.push("-o", outputFormat);
+  if (outputFormat == "html") {
+    cmdArgs.push("-o", "template");
+    cmdArgs.push("-t", "html.tmpl");
```

Comment:
Have you tried this out? I wouldn't think this `html.tmpl` would be placed in the current working directory -- for example running this with `path: .` runs in and resolves to the _workspace root_.

I wonder if this `html` enhancement would be better suited to be added directly to `grype` and then exposed here as an option... I also acknowledge that many people might like different output formats.

What if, instead of including the `html` in _this_ repo, you include the HTML template you would like in your own repo and we change this to a `template` output where you reference the template to use?